pkgs/applications/version-management/gitea/default.nix

```diff
@@ -33,6 +34,12 @@ buildGoPackage rec {
 
   patches = [
     ./static-root-path.patch
+
+    (fetchpatch {
+      name = "go_1_18-support.patch";
+      url = "https://patch-diff.githubusercontent.com/raw/go-gitea/gitea/pull/19100.patch";
```

Comment:
Do we really need this? AFAICS `buildGoPackage` still uses Go 1.17. Also, if you add patches like this, please use URLs pointing to commit hashes rather than PRs, if a new change is pushed to that PR, the hash becomes invalid.

Btw, IIRC we have gitea 1.15 on 21.11, do you think we should just backport 1.16 (despite potential breaking changes) or do you think it's feasible to only backport the relevant patches?

Reply:
Wouldn't `buildGoPackage` be affected when go default changes to 1_18 by default? Although I am happy to change this to be explicitly built by 1_17 instead.

As for backporting, I'd have to check if the patches cleanly apply to 1.15.x before saying one way or the other.

Let me remove this patch from this PR, so at least this update can make it in (as go_1_18 hasn't yet been updated to default).

Reply:
You could also use `buildGo117Package` (not sure about the correct name)